projects/ngx-mapbox-gl/src/lib/marker/marker.component.ts

```diff
@@ -26,6 +26,8 @@ export class MarkerComponent implements OnChanges, OnDestroy, AfterViewInit, OnI
   /* Init input */
   @Input() offset?: PointLike;
   @Input() anchor?: Anchor;
+  @Input() pitchAlignment?: Alignment;
```

Comment:
Why are these properties init-only? As far as I can see, there is API to set them after a marker has been created?

Reply:
I'd also like to see some showcase included

Reply:
Even though the docs mention a `setPitchAlignment` method, this is not present in the typings:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mapbox-gl/index.d.ts#L1429

Since I can't imagine a use case in which the alignment options would change over time, I'd suggest to keep those options init-only for now, at least until someone provides a feasable scenario. We would first have to update the upstream mapbox-gl types though.

As you requested, I added a showcase example. Hope you like it and won't get seasick :)

Reply:
You can easily add it to typing and i'll quickly approve the change (i'm one of owners for those typings)

Reply:
Do not get me wrong, i'm very thankful for your contribution. I just want to do it by the book and not to return to this feature later again. If you need this first right away for your project, you can use a private build in meantime. Thanks.

Reply:
No problem. Like you, I want to do things by the book and not use private forks in my app ;-)

Expect an update on this as soon as the npm package of @types/mapbox-gl is published.